src/components/common/input-container/Container.tsx

```diff
@@ -0,0 +1,23 @@
+import { DetailedHTMLProps, HTMLAttributes, ReactNode } from 'react';
+
+type BaseContainerProps = DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>;
+
+interface ContainerProps extends BaseContainerProps {
+  title?: string;
+  children: ReactNode;
+}
+
+const Container = ({ title, children }: ContainerProps) => {
```

Comment:
아래와 같이 작성되어도 동일할 것 같은데, 타입이 복잡하게 작성된 것 같은데 의도한 바가 있을까요?!

```suggestion
interface ContainerProps extends PropsWithChildren {
  title?: string;
}

const Container = ({ title, children }: ContainerProps) => {
```

Reply:
의도한 바는 없고 타입스크립트 사용이 미숙해서 그렇습니다. 
수정해주신 부분이 간결하고 보기 좋은거 같습니다 !! 
수정하겠습니다!